reporters/GlobalSummaryReporter.js

```diff
@@ -346,7 +346,8 @@ class GlobalSummaryReporter {
         }
       }
 
-      // Create list of resolved prices for disputed liquidations.
+      // Create list of resolved prices for disputed liquidations. Note that this `web3.getBlock().timestamp` call to get the liquidation timestamp
+      // only works on public networks. It will NOT work on local networks that use the MockOracle/Timer contract where block timestamp !== EMP timestamp.
       const liquidationTimestamp = (await this.web3.eth.getBlock(liquidationData.blockNumber)).timestamp;
```

Comment:
This is also hacky because we assume that the liquidation's block timestamp is equal to the `liquidationTime` prop in the liquidation struct. This would NOT be the case if the liquidation were done on a local network using the `Timer` contract that allows EMP contract time to differ from block time.